ci-operator/step-registry/storage/conf/wait-for-csi-driver/storage-conf-wait-for-csi-driver-commands.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+set -x
+
+echo "Waiting for the ClusterCSIDriver $CLUSTERCSIDRIVER to get created"
+while true; do
+    oc get clustercsidriver $CLUSTERCSIDRIVER -o yaml && break
+    sleep 5
+done
+
+ARGS=""
+for CND in $TRUECONDITIONS; do
+    ARGS="$ARGS --for=condition=$CND"
+done
+
+echo "Waiting for the ClusterCSIDriver $CLUSTERCSIDRIVER conditions $ARGS"
+if ! oc wait --timeout=300s $ARGS clustercsidriver csi.vsphere.vmware.com; then
```

Comment:
nit - might as well use `$CLUSTERCSIDRIVER`  rather than hardcoding the name?

Reply:
good catch!

Reply:
fixed